src/api/operations/bulkService.queries.graphql

```diff
@@ -22,6 +22,13 @@ query BulkServicesClientSearch(
       id
       ...ClientName
       ...ClientIdentificationFields
+      alerts {
+        ...ClientAlertFields
+      }
+      access {
+        canViewClientAlerts
+        canManageClientAlerts
```

Comment:
I thought it was a bit silly to add `canManageClientAlerts` here to keep typescript happy, even though the user is never actually managing client alerts from the bulk services page. I think that the juice is probably not worth the squeeze to avoid it? But let me know if I should do so.

Reply:
I don't think it should be necessary to resolve either of these. The `alerts` array will be empty if the user does not have permission to view alerts, that is already checked [here](https://github.com/greenriver/hmis-warehouse/blob/324fbfe9a2d4e2b748ca0a7ece6e4c402c1dd46c/drivers/hmis/app/graphql/types/hmis_schema/client.rb#L331-L335).

In some cases we check on the frontend because the UX is different based on whether the user has access or not. I don't think that's the case here.

Reply:
ts to make access object optional can be:
`Omit<ClientWithAlertFieldsFragment, 'access'> & Partial<ClientWithAlertFieldsFragment>;`

Reply:
Ah perfect. Thank you for the suggestion 👍